src/VoucherBundle/Controller/BookletController.php

```diff
@@ -381,7 +381,7 @@ public function updatePasswordAction(Request $request)
 
     /**
      * Assign the booklet to a specific beneficiary
-     * @Rest\Post("/booklets/assign/{beneficiaryId}/{distributionId}", name="assign_booklet")
+     * @Rest\Post("/booklets/assign/{distributionId}/{beneficiaryId}", name="assign_booklet")
```

Comment:
I think you need to change the order of the parameters line 403 to match with the order of the parameters